dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/common/info/OpenTelemetryInfoFactoryImpl.java

```diff
@@ -75,7 +75,7 @@ public class OpenTelemetryInfoFactoryImpl implements ApplicationStateListener, O
     private static final TraceComponent tc = Tr.register(OpenTelemetryInfoFactoryImpl.class);
 
     private final MetaDataSlot slotForOpenTelemetryInfoHolder;
-    Map<String, OpenTelemetry> otelMap = new HashMap<String, OpenTelemetry>();
+    private EnabledOpenTelemetryInfo runtimeInstance = null;
```

Comment:
Normally I would say store by interface not by the concrete type. But in this case I think its correct to say this is the only valid impl.

Reply:
Yeah, I had originally added this to prevent a new object from being created each time the runtime instance was requested.